CONTRIBUTING.md

```diff
@@ -7,7 +7,7 @@ This project is a complete rewrite and replacement for mechas2's codebase, writt
 Please note that while some organization members have direct write access, all code additions are **REQUIRED** to be peer reviewed.
 
 ## Getting Started
-
+0. Check with the mecha3 team lead prior starting work on a feature/change. 
```

Comment:
[Question] Since the project is known to GitHub as pipsqueak3, should we not use that name here as well?

Additionally, is there any way for an outsider to easily determine who the lead is?

Reply:
technically speaking we should use `Spark` since that's the formal project identifier, and thats what is front-and-center on the README.

Reply:
Language updated.